packages/evm-connector/signature-verification/Forc.toml

```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["verification-predicate", "verification-script"]
```

Comment:
We should remove the `"verification-script"` problably and only keep the predicate code itself, and with this all references to the script.